src/CoreWCF.Primitives/src/CoreWCF/Description/ServiceDescription.cs

```diff
@@ -333,5 +351,20 @@ public ServiceDescription(IEnumerable<IServiceBehavior> injectedBehaviors, IServ
             AddBehaviors<TService>(this, behaviors);
             SetupSingleton<TService>(this, services);
         }
+
+        private static Lazy<Func<IServiceProvider, IEnumerable<IServiceBehavior>>> s_getKeyedServiceBehaviorsDelegate = new(() =>
+        {
+            Assembly assembly = Assembly.Load("Microsoft.Extensions.DependencyInjection.Abstractions");
```

Comment:
Is this assembly guaranteed to be available at runtime? Or do we depend on the developer having added a package?

Reply:
`CoreWCF.Primitives` pulls it and it also comes with `Microsoft.NET.Sdk.Web`.